dynalab/tasks/annotation_mock_data.py

```diff
@@ -0,0 +1,78 @@
+import random
+
+from dynalab.tasks.annotation_types import AnnotationTypeEnum
+
+
+def get_source_data(annotation, name_to_annotation_dict):
+    source_reference_name = annotation["constructor_args"]["reference_name"]
+    source_annotation = name_to_annotation_dict[source_reference_name]
+    source_data = annotation_mock_data_generators[source_annotation["type"]](
+        source_annotation, name_to_annotation_dict
+    )
+    return source_data
+
+
+def generate_image_url_mock_data(annotation, name_to_annotation_dict):
+    return ["https://dynabench.org/logo_w.png"]
+
+
+def generate_string_mock_data(annotation, name_to_annotation_dict):
+    return [
+        "It is a good day",
+        "Let's try a utf-8 like hackamore from j?\u00a1quima;",
+        " ".join([str(x) + "_" for x in range(513)]),
+    ]
+
+
+def generate_context_string_selection_mock_data(annotation, name_to_annotation_dict):
+    source_data = get_source_data(annotation, name_to_annotation_dict)
+    return [source_str[0:10] for source_str in source_data]
+
+
+def generate_conf_mock_data(annotation, name_to_annotation_dict):
+    return [random.random() for _ in range(3)]
+
+
+def generate_multiclass_probs_mock_data(annotation, name_to_annotation_dict):
+    mock_data = []
+    source_reference_name = annotation["constructor_args"]["reference_name"]
+    source_annotation = name_to_annotation_dict[source_reference_name]
+    labels = source_annotation["constructor_args"]["labels"]
+
+    for _ in range(3):
```

Comment:
Why 3?

Reply:
We originally had 3 types of samples for string input type, so I thought of keeping it consistent in a way that all the mock inputs generated by us can have 3 random samples. What do you think is the best way?

Reply:
Hmm it should depend on how many mock data points are there? The idea was to support multiple test input to cover various cases and it just happened that we covered three types (which we've seen mistakes) ;)

Reply:
Hmmm wait why do we need mock data for probability? Isn't that part of the model output?

Reply:
We discussed offline that these functions are required for 2 cases - 
1. In principle, any i/o type can be used as a model input, so we might need these methods in the future when i/o types such as `conf` or `prob` might be used as inputs.
2. More importantly, these methods are reused to create sample outputs.